packages/atomic/src/components/commerce/atomic-commerce-search-box/atomic-commerce-search-box.tsx

```diff
@@ -92,7 +94,7 @@ import {SelectChildProductEventArgs} from '../product-template-components/atomic
  * @part instant-results-show-all - The clickable suggestion to show all items for the current instant results search rendered by an `atomic-commerce-search-box-instant-products` component.
  * @part instant-results-show-all-button - The button inside the clickable suggestion from the `atomic-commerce-search-box-instant-products` component.
  *
- * @internal
+ * @alpha
```

Comment:
@jpmarceau would there be an easy way to 'hide' or put annotation on doc pages when there's some specific JSDOC tag?

 Otherwise, I'll try again to filter'em'out differently on my side

Reply:
Morning @louis-bompart 
Good question, thanks for checking.
Yeah, I think we could implement that fairly quickly. Feel free to use the `alpha` annotation and I'll handle them docs-side for the next release 👍

Reply:
What's the reason for the tag change ?

Reply:
This is due to the generation of the Custom Element Manifest.
`@internal` impacts the visibility modifier (private/public) of the AST handling of the tool. I struggled to wrangle the tool to make it generate the manifest for specific internal tags.

Without a proper manifest, Storybook params handling ain't right, which in turn make the param setting in the test not right either